BDArmory/FX/ExplosionFX.cs

```diff
@@ -638,7 +639,10 @@ private void ExecutePartBlastEvent(PartBlastHitEvent eventToExecute)
             var realDistance = eventToExecute.Distance;
             var vesselMass = part.vessel.totalMass;
             if (vesselMass == 0) vesselMass = part.mass; // Sometimes if the root part is the only part of the vessel, then part.vessel.totalMass is 0, despite the part.mass not being 0.
-
+	    var cumulativeHPOfIntermediateParts = eventToExecute.IntermediateParts.Select(p => p.Item2).Sum();
+            var cumulativeArmorOfIntermediateParts = eventToExecute.IntermediateParts.Select(p => p.Item3).Sum();
+            if (realDistance <= Range) //within radius of Blast
+            {
             if (!eventToExecute.IsNegativePressure)
```

Comment:
standard blast damage stuff happening to parts within the blast range, so avoid having to make additional blastUtils calcs for stuff that would be out of range